appeals/web/src/server/appeals/appeal-details/hearing/setup/set-up-hearing.mapper.js

```diff
@@ -0,0 +1,88 @@
+import { appealShortReference } from '#lib/appeals-formatter.js';
+
+/**
+ * @typedef {import('../../appeal-details.types.js').WebAppeal} Appeal
+ */
+
+/**
+ * @param {Appeal} appealData
+ * @param {{ day: string, month: string, year: string, hour: string, minute: string }} values
+ * @returns {PageContent}
+ */
+export function setUpHearingPage(appealData, values) {
+	const shortAppealReference = appealShortReference(appealData.appealReference);
+	const date = { day: values.day || '', month: values.month || '', year: values.year || '' };
+	const time = { hour: values.hour || '', minute: values.minute || '' };
+
+	/** @type {PageComponent} */
+	const dateComponent = {
```

Comment:
It's not strictly necessary to use it, but there's a helper for this component in `appeals/web/src/server/lib/mappers/components/page-components/date.js` which might be useful. Feel free to update it to support your use case if required. Also might be worth adding a similar helper for the time input, in the spirit of continuous improvement